libs/wingsdk/test/target-sim/bucket.test.ts

```diff
@@ -225,7 +294,10 @@ test("remove object from a bucket with mustExist as option", async () => {
 
   // GIVEN
   const app = new SimApp();
-  cloud.Bucket._newBucket(app, bucketName);
+
+  const bucket = cloud.Bucket._newBucket(app, bucketName);
+  const testInflight = new InflightBucketEventHandler(app, "inflight_test");
+  bucket.onDelete(testInflight);
```

Comment:
Would prefer moving this into a different test case (e.g. "test that onDelete works") since this test case seems to be about the "mustExist" option